06_miscellaneous/floss_licenses_slides.md

```diff
@@ -0,0 +1,173 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+    font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+  .reveal code {
+    font-family: 'Ubuntu Mono';
+    color: orange;
+  }
+  .reveal section img {
+    background:none;
+    border:none;
+    box-shadow:none;
+  }
+</style>
+
+# FLOSS Licenses
+
+---
+
+## Disclaimer
+
+- IANAL: _I am not a lawyer._
```

Comment:
```suggestion
- IANAL: *I am not a lawyer.*
```
Maybe we should stick to asteriks as emphasis symbol?

Reply:
Yes, we defined this once. I forgot.